labelbox/schema/model_run.py

```diff
@@ -175,6 +181,60 @@ def delete_model_run_data_rows(self, data_row_ids):
             data_row_ids_param: data_row_ids
         })
 
+    @experimental
+    def export_annotations(self, download: bool = False) -> LabelGenerator:
```

Comment:
Let's call this export_labels to be consistent with the project export signature

Reply:
and change the return type to `Optional[Union[str, List[Dict[Any, Any]]]]`